business/sys/database/database.go

```diff
@@ -147,6 +155,10 @@ func NamedExecContext(ctx context.Context, log *zap.SugaredLogger, db sqlx.ExtCo
 	log.Infow("database.NamedExecContext", "traceid", web.GetTraceID(ctx), "query", q)
 
 	if _, err := sqlx.NamedExecContext(ctx, db, query, data); err != nil {
+		// Checks if the error is of code 23505 (unique_violation).
```

Comment:
Comments require a line space above for readability.

Reply:
done!